Resources/translations/fr/section.fr.xlf

```diff
@@ -0,0 +1,113 @@
+                <source>section.name.label</source>
+                <target>Nom de section:</target>
+            </trans-unit>
+            <trans-unit id="3de80c254c9555ad1347b034f93f3095" resname="section.identifier.label">
+                <source>section.identifier.label</source>
+                <target>Identifiant de section:</target>
+            </trans-unit>
+            <trans-unit id="f066f9960df2f81907c3680c9148cef5" resname="section.id.label">
+                <source>section.id.label</source>
+                <target>ID de section:</target>
+            </trans-unit>
+            <trans-unit id="751bbc52342db46a046c52c4dc0b6b00" resname="section.label.edit">
+                <source>section.label.edit</source>
+                <target>Éditer la section</target>
+            </trans-unit>
+            <trans-unit id="60ff6d8b447be276d65797a5244d79c3" resname="section.error.id_already_exist">
+                <source>section.error.id_already_exist</source>
+                <target>Une section avec cet identifiant existe déjà, merci de saisir un autre identifiant</target>
+            </trans-unit>
+            <trans-unit id="85af0dc2a9b83eb5cfc7871a73cbeab9" resname="section.create.submit">
+                <source>section.create.submit</source>
+                <target>Sauvegarder</target>
+            </trans-unit>
+            <trans-unit id="94bfbed9c62011b82e87a97ea9ea9499">
+                <source>section.cancel</source>
+                <target>Annuler</target>
+            </trans-unit>
+            <trans-unit id="22e8e66a091b0c1227c4461e51e06443">
+                <source>section.deleted</source>
+                <target>La Section "%sectionName%" a été correctement supprimée.</target>
+            </trans-unit>
+            <trans-unit id="1bf9bdce5e87c98df324d43e717d9d85" resname="section.assign.universaldiscovery.title">
+                <source>section.assign.universaldiscovery.title</source>
+                <target>Choisir le contenu assigné à la section "%sectionName%"</target>
+            </trans-unit>
+            <trans-unit id="058b034a4aa957ab59f853e8375ff2b3" resname="section.assign.contents">
+                <source>section.assign.contents</source>
+                <target>Assigner au contenu</target>
+            </trans-unit>
+            <trans-unit id="419a9a2e9f888d56b6aea6e4cbb09561">
+                <source>section.notification.created</source>
+                <target>La section "%sectionName%" a été correctement créée.</target>
+            </trans-unit>
+            <trans-unit id="b49dc3557e76c56e175d0e133afa05d4">
+                <source>section.notification.updated</source>
+                <target>La section "%sectionName%" a été correctement sauvegardée.</target>
+            </trans-unit>
+            <trans-unit id="b93c76061ec3d7810f4175b3681a049e">
+                <source>section.notification.draft_removed</source>
```

Comment:
@andrerom same here, as far as I know we don't have draft for section, so do we have such string ?